JetSet/JetSet/Models/AircraftProfile.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AircraftProfile.swift
+//  JetSet
+//
+//  Created by Trevor Sharnick on 12/10/23.
+//
+
+import Foundation
+
+/// AircraftProfile is the model which represents the information displayed on each profile card. It also contains the aircraft's checklist information available when pressing on the corresponding profile card
+struct AircraftProfile: Codable, Identifiable {
+    let id = UUID()
+    var title: String
+    var image: String?
+    var checklists: [Checklist]
+    
+    var imageURL: URL? {
+        guard let image else {
+            return nil
+        }
+        return URL(string: image)
+    }
+    
+    enum CodingKeys: String, CodingKey {
+        case title
+        case image
+        case checklists
+    }
+    
+} 
+
+extension AircraftProfile {
+    static let preview = AircraftProfile(title: "A-10 Warthog",
+                                         image: Bundle.main.url(forResource: "Warthog", withExtension: "jpg")!.absoluteString,
+                                         checklists: [])
+    
+    static let noImage = AircraftProfile(title: "A-10 Warthog", image: nil, checklists: [])
+}
```

Comment:
Thanks Nate, will do this tonight.

Reply:
Looks like you created a new branch to do these changes based off this branch and then merged that branch, which resulted in this branch ending up merged as well:
<img width="804" alt="Screenshot 2023-12-11 at 11 53 06 PM" src="https://github.com/TrevorSharnick/JetSetApp/assets/4147804/947657ce-862d-4c18-8d02-b2cb1e6aced6">

Instead of opening a new branch, it's standard practice to just commit those changes to this branch, and then push those changes to this branch so that they appear as commits on this PR